README.md

```diff
@@ -1 +1,21 @@
-# eth-bucharest
\ No newline at end of file
+# Initial setup guide
+
+1. Clone this repo from github
+2. On the root directory, create an `.env` file and copy the contents of the [env.example](.env.example). Fill out the necessary variables.
+
+> This project runs on a specific version of yarn - @4.1.1. If you are already on this version of yarn, you may skip directly to next section
+
+3. If you are using a different version, consider using **nvm** to handle different version of node and yarn on your computer.
+4. If you already have nvm, please skip to next step. If you do not already have nvm installed, you can follow a more detailed instructions here on [nvm github repo](https://github.com/nvm-sh/nvm).
+5. Once you have installed and verified your nvm installation, you can now install `yarn`.
+6. First enable corepack using `corepack enable`
```

Comment:
I would put a warning before step 6:
Yarn will be installed globally so make sure it doesn't affect other projects which require a different version of yarn. By using nvm, each node version you install will have it's own global packages so you can change to different versions of yarn.